ui/app/templates/partials/secret-list/transform-transformation-item.hbs

```diff
@@ -0,0 +1,62 @@
+{{#linked-block
+  (concat
```

Comment:
Probably don't need concat if it's all magic strings

Reply:
Good catch, another carry over from copying from a different template.